### PR TITLE
Update for version 2068.

### PR DESCRIPTION
**Pull request by ryanpcmcquen**

### CODE REVIEW

**Comment by ryanpcmcquen** [comment]
Closes:
https://github.com/flathub/com.sublimemerge.App/issues/28

File: com.sublimemerge.App.metainfo.xml [hunk 1]
```diff
@@ -25,6 +25,43 @@
...
   </screenshots>
   <releases>
+    <release date="2021-12-21" version="0.2068">
+      <p>BUILD 2068</p>
+      <h3>New Features and Improvements</h3>
+      <ul>
+          <li>Search: Quickly write queries with auto-complete suggestions</li>
+          <li>Improved OpenGL rendering performance by automatically batching together controls</li>
+          <li>Fetch / Pull Command: Added ability to fetch tags</li>
+          <li>Merge Command: Merge unrelated histories using the <tt>allow-unrelated-histories</tt> flag</li>
```
Comment (ptomato) [inline com.sublimemerge.App.metainfo.xml]: It looks like the `<tt>` tag is not supported here and that's why the build is failing. Could you remove the `<tt>` and `</tt>` from around the text?

**Review by ptomato — APPROVED** [review]
Thanks! As soon as we can get this to build, I think it's good to go.

File: com.sublimemerge.App.metainfo.xml (hunk 1 above)
Comment (ryanpcmcquen) [reply com.sublimemerge.App.metainfo.xml]: Good eye! Let's see if I got them all. 🤞

**Comment by ptomato** [comment]
Works for me. Thanks!

**Pull request merged**